lib/screens/surveydetails/widget/question_paging_widget.dart

```diff
@@ -153,6 +154,8 @@ class _QuestionPagingWidgetState extends State<QuestionPagingWidget> {
 
   Widget _buildAnswerWidget(QuestionUiModel question) {
     switch (question.displayType) {
+      case DisplayType.nps:
+        return _buildNpsRatingWidget(question);
```

Comment:
Optional: 

```suggestion
        return _buildAnswerNpsRatingWidget(question);
```

Just for consistency.

Reply:
Updated in [ef138a6](https://github.com/nimblehq/flutter-ic-lydia-ryan/pull/93/commits/ef138a62513adb3765f5d8f316fcc3d98e6ff196)